addons/mail/models/mail_channel.py

```diff
@@ -731,15 +732,12 @@ def channel_seen(self):
                 'fetched_message_id': last_message_id,
             })
             data = {
+                'id': channel_partner.id,
                 'info': 'channel_seen',
                 'last_message_id': last_message_id,
                 'partner_id': self.env.user.partner_id.id,
             }
-            if self.channel_type == 'chat':
-                self.env['bus.bus'].sendmany([[(self._cr.dbname, 'mail.channel', self.id), data]])
-            else:
-                data['channel_id'] = self.id
-                self.env['bus.bus'].sendone((self._cr.dbname, 'res.partner', self.env.user.partner_id.id), data)
+            self.env['bus.bus'].sendmany([[(self._cr.dbname, 'mail.channel', self.id), data]])
```

Comment:
I don't understand the question ?

Reply:
You made that change to support it on channels to be able to test but that was disabled for perf reasons, we must not reintroduce it in this PR.

Reply:
Ah, didn't understand that at first. I'll revert that then but we need to handle things in templates because seen indicators will be seen no matter what ..